### PR TITLE
Added Twitter authorization and notifications

### PR DESCRIPTION
**Pull request by khellang**
It would be very nice if some of you could test it out :) Please tell if the code is totally off...

### CODE REVIEW

File: app/lib/twitter.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    def get_authorization(self):
+
+        oauth_consumer = oauth.Consumer(self.consumer_key, self.consumer_secret)
+        oauth_client = oauth.Client(oauth_consumer)
+
+        log.info('Getting authentication url from Twitter')
+
+        host = cherrypy.config['config'].get('global', 'host')
+        port = cherrypy.config['config'].get('global', 'port')
+        callbackURL = 'http://'+host+':'+port+'/config/twitterAuth/'
```
Comment (khellang) [inline app/lib/twitter.py:74]: This is probably not the best way to generate the callbackURL. There has to be a way to get the host dynamicly.

**Comment by RuudBurger** [comment]
This also contains stuff for the NZBGet merge, could you remove those? 
Also is the simplejson needed? As python 2.6 has a default json class.

**Comment by khellang** [comment]
Yeah, I noticed the nzbget part, I'll remove it. The simplejson is a dependency of python-twitter. If you want to remove it someone will have to write another Twitter lib...

**Comment by RuudBurger** [comment]
in library/pythontwitter/**init**.py  on line nr 39. It loads the default json ;)
So the library isn't needed.

**Comment by khellang** [comment]
Now the lib is removed. I tested without it, and it worked :) Dunno where i got that it was a python-twitter dependency? :S

**Comment by RuudBurger** [comment]
Awesome! Thanks, I know a few people who gonna like this ;)

File: app/lib/cron/renamer.py [hunk 2]
```diff
@@ -182,6 +183,11 @@ class RenamerCron(cronBase, Library):
                 nma = NMA()
                 nma.notify('Download Complete', 'Downloaded %s (%s)' % (movie['movie'].name, movie['movie'].year))
 
+                # Notify Twitter
+                log.debug('Twitter')
+                twitter = Twitter()
+                twitter.notify('Downloaded %s (%s)' % (movie['movie'].name, movie['movie'].year), 'Downloaded:')
```
Comment (khellang) [inline app/lib/cron/renamer.py:189]: I just noticed; either these two parameters have to be switched or line 53 in twitter.py have to be changed to include event also...

Comment (RuudBurger) [inline app/lib/cron/renamer.py:189]: Do your thing, I'm not a "coding" computer right now.
Not a big bug though, can wait if you don't have the time. If you do, just pull request it!

Thanks!

**Pull request merged**